admin/endpoints/class-endpoint-indexable.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Endpoints
+ */
+
+/**
+ * Represents an implementation of the WPSEO_Endpoint interface to register one or multiple endpoints.
+ */
+class WPSEO_Endpoint_Indexable implements WPSEO_Endpoint, WPSEO_Endpoint_Storable {
+
+	const REST_NAMESPACE = 'yoast/v1';
+	const ENDPOINT_RETRIEVE = 'indexables/(?P<object_type>.*)/(?P<object_id>\d+)';
+
+	const CAPABILITY_RETRIEVE = 'manage_options';
+	const CAPABILITY_STORE = 'manage_options';
```

Comment:
I wonder whether we could be more specific and granular with these capabilities. To be fair though, that might be a change better served through a separate ticket, as there are several locations in the plugin where that could be improved.